curriculum/challenges/english/08-coding-interview-prep/rosetta-code/semiprime.english.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 5eb3e4aa847216613aa81983
+title: Semiprime
+challengeType: 5
+forumTopicId: 385318
+---
+
+## Description
+
+<section id='description'>
+Semiprime numbers are natural numbers that are products of exactly two (possibly equal) <a href="http://rosettacode.org/wiki/prime_number" target="_blank">prime numbers</a>.
+ 1679  =  23 &times; 73
+</section>
+
+## Instructions
+
+<section id='instructions'>
+Write a function determining whether a given number is semiprime.
```

Comment:
Write a function that returns true if a number is semiprime, or false if it is not.